.github/workflows/tests_lgpumpi_cpp.yml

```diff
@@ -44,6 +44,7 @@ jobs:
       max-parallel: 1
       matrix:
         mpilib: ["mpich", "openmpi"]
+        enable_lapack: ["OFF", "ON"]
```

Comment:
This workflow is already quite demanding, do we really need to test both with and without LAPACK given that the non-MPI code already tests that.